plugins/connection/pct_remote.py

```diff
@@ -0,0 +1,772 @@
+# -*- coding: utf-8 -*-
+# Derived from ansible/plugins/connection/paramiko_ssh.py (c) 2012, Michael DeHaan <michael.dehaan@gmail.com>
+# Copyright (c) 2024 Nils Stein (@mietzen) <github.nstein@mailbox.org>
+# Copyright (c) 2024 Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (annotations, absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+    author: Nils Stein (@mietzen) <github.nstein@mailbox.org>
+    name: pct_remote
+    short_description: Run tasks in Proxmox LXC container instances using pct CLI via SSH
+    requirements:
```

Comment:
Please remove this 4-space indentation from the top level, it is inconsistent with most of the plugins (no indentation for the top level) and in this case it is also inconsistent with the rest of the YAML block (2 spaces).

Reply:
Done

Reply:
I have been actually normalizing the doc blocks everywhere and, to be fair, connection plugins were looking exactly like that, but PR has just been merged to adjust it. Thanks anyways for making the change here.